packages/mds-transaction-api/spec/flat-spec.json

```diff
@@ -0,0 +1,674 @@
+{
```

Comment:
Same comment applies here if the files in `spec` are generated as well.

Reply:
Fixed in e730cd9